app/presenters/task_presenter/completed.rb

```diff
@@ -1,16 +1,28 @@
 module TaskPresenter
   class Completed < SimpleDelegator
 
-    def cancellation_completion_command
-      TaskCancellationCompletion.new(task_id: id)
-    end
-
-    def completed_on
-      completed_task.completed_at.to_date
+    def timestamp
+      I18n.l(completed_on, format: :default)
     end
 
     def completion_checkbox_state
       %(checked="checked").html_safe
     end
+
+    def completion_form(view)
+      view.form_for(cancellation_completion_command, url: view.cancel_completion_task_url(self), method: :put, remote: true) do |f|
```

Comment:
__[rubocop]__
* Lint/UnusedBlockArgument: Unused block argument - `f`. You can omit the argument if you don't care about it. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)